fgd/point/env/env_global.fgd

```diff
@@ -34,7 +34,7 @@
 
 	globalstate[INFRA](choices) : "Global State to Set" =
 		[
-		"infra_no_tools": "INFRA: player doesn't start with his tools"
+		"infra_no_tools": "INFRA: player doesn't start with their tools"
```

Comment:
Might want to keep these male pronouns, since Mark is a voiced character, the credit card has his name on it etc.